xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1404,19 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last incremental profile stat started and last
+       fop hit.
+    */
+    duration = (conf->last_fop_hit - stats->started_at);
+    if (duration > 0) {
```

Comment:
I think we should always report the value, even if it's 0. Also consider that `duration` has a resolution of seconds, so there are cases where it could be 0, but some time may have actually passed and some fops executed (so `total_fop_hits` will be > 0). In this case we should probably show the fops/s as if `duration` was 1

Reply:
done